### PR TITLE
Fixing some issues related to vEBTree.grow(self, to_size) function.

### PR DESCRIPTION
**Pull request by WizardPlatypus**
I've run into a few problems while working with veb pip package, so I decieded to fix them.

The main problems were:
1) Using `vEBTree.grow(self, to_size)` function would reset min and max properties, which would result in lose of data.
2) `vEBTree.update(self, iterable)` function didn't use to always add all the elements from iterable to the tree.
3) `vEBTree.update(self, iterable)` would call `vEBTree.grow(self, to_size)` function multiple times, while calling it only once is necessary.

I added some tests to adress those issues, then written some code that solves them — tests are passing.

One major problem with my pullrequest: I couldn't get tox to see tests I added, so I used basic `python -m unittest discover -v` method, which worked. So tests are there, they're passing, but tox refuses to see them, as far as I understand.

### CODE REVIEW

**Comment by Julian** [comment]
Nice! Thanks! I haven't looked at this library in ages :) but will have a look, really appreciate the PR.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+# [WizardPlatypus]: the only modules used by tests are from standard library, so I assume this file should be empty.
```
Comment (Julian) [inline test-requirements.txt:1]: Yes which means there's no need to include it essentially.

**Comment by Julian** [comment]
Thanks again! Merged (and then did a bunch of random packaging modernization). Appreciated, let me know if you find any other issues :)

**Comment by WizardPlatypus** [comment]
> Thanks again! Merged (and then did a bunch of random packaging modernization). Appreciated, let me know if you find any other issues :)

Happy to help, will surely do = ]

**Pull request merged**